### PR TITLE
Added SubstFor

### PR DESCRIPTION
**Pull request by Abdullahjavednesar**

### CODE REVIEW

**Comment by arihantparsoya** [comment]
The tests are passing?

**Comment by arihantparsoya** [comment]
Implement remaining functions also

**Comment by arihantparsoya** [comment]
Add `RectifyTangent`,`RectifyCotangent`

**Comment by arihantparsoya** [comment]
Add `FunctionOfTrigOfLinearQ`

**Comment by arihantparsoya** [comment]
Add `FresnelS`, `FresnelC`, `Gamma`, `Erfc` , `Erfi`.

Refer to `special function` rules.

**Comment by arihantparsoya** [comment]
Add `PolyLog`

**Comment by Abdullahjavednesar** [comment]
@parsoyaarihant, you have already implemented `RectifyTangent`,`RectifyCotangent`, complete those  I'll complete the rest.

File: sympy/integrals/rubi/utility_function.py [hunk 1]
```diff
@@ -17,12 +17,14 @@
...
 from sympy.utilities.iterables import postorder_traversal
 from sympy.core.expr import UnevaluatedExpr
+from sympy.functions.special.error_functions import fresnelc, fresnels
 from sympy.functions.elementary.complexes import im, re, Abs
 from sympy.core.exprtools import factor_terms
 from sympy import (exp, polylog, N, Wild, factor, gcd, Sum, S, I, Mul, Add, hyper,
     Symbol, symbols, sqf_list, sqf, Max, gcd, hyperexpand, trigsimp, factorint,
     Min, Max, sign, E, expand_trig, poly, apart, lcm, And, Pow, pi, zoo, oo)
-from mpmath import appellf1
+from mpmath import appellf1, erfi, erfc, gammainc, polylog
```
Comment (arihantparsoya) [inline sympy/integrals/rubi/utility_function.py]: Don't use mpmath. Use sympy functions.

**Comment by arihantparsoya** [comment]
Can you complete these lines: https://github.com/parsoyaarihant/sympy/pull/16/files/de16c030a0300519a88b75dc4a08e1b3ac27971f..4b3f559c01eaf569eedb8c4c65462a817b78b666#diff-7155fe6e5fd15253f50ed2c086d5354aR5676

File: sympy/integrals/rubi/utility_function.py (hunk 1 above)
Comment (Abdullahjavednesar) [reply sympy/integrals/rubi/utility_function.py]: Does sympy has equivalent functions?

Comment (arihantparsoya) [reply sympy/integrals/rubi/utility_function.py]: yes

**Pull request merged**